form.yml.erb

```diff
@@ -7,6 +7,9 @@
 cluster:
   - "owens"
   - "pitzer"
+<% if OodAppkit.clusters.key?("kubernetes") && OodAppkit.clusters["kubernetes"].allow? -%>
```

Comment:
I don't know enough about how these forms are processed to know if maybe OnDemand automagically will just ignore a defined cluster in the form if that cluster is not defined or not allowed.  Given I don't know the behavior here inside the form handling, I opted for what is the safest approach of just not defining the cluster if it's not defined in `clusters.d` and not allowed by a given user.